.github/workflows/build_release.yaml

```
@@ -0,0 +1,66 @@
+name: Rust Build and Release
```

```suggestion
name: Release
```

I would like to make it simpler 🙏